src/networkAddresses.ts

```diff
@@ -109,6 +118,9 @@ export const NETWORK_ADDRESSES = {
     RESILIENT_ORACLE: "0x8f3618c4F0183e14A218782c116fb2438571dAC9",
     VTREASURY: "0xDDc9017F3073aa53a4A8535163b0bf7311F72C52",
     BINANCE_ORACLE: "0xB09EC9B628d04E1287216Aa3e2432291f50F9588",
+    ENDPOINT: "0xb6319cC6c8c27A8F5dAF0dD3DF91EA35C4720dd7",
+    LZ_LIBRARY: "0x38dE71124f7a447a01D67945a51eDcE9FF491251",
+    OMNICHAIN_GOVERNANCE_EXECUTOR: "",
   },
   arbitrumsepolia: {
     NORMAL_TIMELOCK: "0x1426A5Ae009c4443188DA8793751024E358A61C2", // arbitrumsepolia Multisig
```

Comment:
Shouldn't we add `LZ_LIBRARY`, `ENDPOINT` and `OMNICHAIN_GOVERNANCE_EXECUTOR` address for arbitrumsepolia also?

Reply:
We will configure arbitrumSepolia in another VIP once we are done with all final deployments on arbitrum.